application/src/main/java/net/furizon/backend/service/users/UserService.java

```diff
@@ -69,12 +63,13 @@ public void login(
             body.getEmail(),
             body.getPassword()
         );
-        Authentication authentication = authenticationManager.authenticate(token);
-        SecurityContextHolderStrategy securityContextHolderStrategy = SecurityContextHolder.getContextHolderStrategy();
-        SecurityContext context = securityContextHolderStrategy.createEmptyContext();
-        context.setAuthentication(authentication);
-        securityContextHolderStrategy.setContext(context);
-        securityContextRepository.saveContext(context, request, response);
+        //Authentication authentication = authenticationManager.authenticate(token);
```

Comment:
This thing should not be in Service, it supposed to be on the authentication filter chain; by login method, we should create a session/token and operate with it